tests/generators/OperationGenerator.ts

```diff
@@ -549,7 +586,7 @@ export default class OperationGenerator {
     if (idOfServiceEndpointToAdd !== undefined) {
       const patch = {
         action: 'add-service-endpoints',
```

Comment:
It would be sensible for the action string to be `add-services` to be consistent with the rest of the rename.